resources/resources.xml

```diff
@@ -6,8 +6,11 @@
 		  <label id="label" x="center" y="bottom"
 				font="Graphics.FONT_SYSTEM_XTINY"
 				justification="Graphics.TEXT_JUSTIFY_CENTER"/>
-			<label id="value" x="center" y="center"
+		  <label id="value" x="center" y="center"
 				font="Graphics.FONT_SYSTEM_NUMBER_THAI_HOT"
 				justification="Graphics.TEXT_JUSTIFY_CENTER"/>
-		</layout>
+		  <label id="mark" x="center" y="top"
+				font="Graphics.FONT_SYSTEM_XTINY"
```

Comment:
Default is FONT_TINY for label and FONT_LARGE for value. Otherwise this datafield stands out from the rest of the UI.
Also on the watch the label sits above the value, seems there should be multiple layouts... E.g. https://github.com/creacominc/connectiq-PowerField/blob/master/resources/layouts/layouts.xml